### DISCUSSION TITLE
1

### DISCUSSION THREAD

**Discussion by dedao168** [filecoin-project/community#450] [Market]
1

**Comment by kaitlin-beegle** [comment]
Hi @dedao168 - do you have a question that needs answering?

**Comment by dedao168** [comment]
Hello，Protocol Lab Team!
Thank you for reading our email. We are Acrontech, A tech incorporation
registered in Delaware, USA. Focus on IPFS distributed storage and Filecoin
mining services. We use VMware Cloud Foundation infrastructures to realize
multiple-nodes, large-scaled and global deployment and maintenance, and
Crew members come from the USA, Canada, and China, including early Filecoin
investors and experts in the IT industry. We already have established the
fundamentally technical preparation and commercial road map were, and we
have already donated more than 5M USD self-owned funds. Now we have
contacted several international institutions that plan to have IPO in
NASDAQ or NYSE in 2023/2024
We require technical and commercial supports and building a long-term
strategic cooperation relationship with you and the community. The
following attachment is our details and business plan. Hope you reply to us
as soon as possible!


Thanks！
                                                          Acrontech Inc.
Team

Kaitlin Beegle ***@***.***> 于2022年1月18日周二 08:19写道：
